h/migrations/versions/e10ce4472966_add_index_to_group_readable_by.py

```diff
@@ -0,0 +1,23 @@
+"""
+Add index to group.readable_by
+
+Revision ID: e10ce4472966
+Revises: f0f42ffaa27d
+Create Date: 2016-12-22 16:13:53.658938
+"""
+
+from __future__ import unicode_literals
+
+from alembic import op
+
+
+revision = 'e10ce4472966'
+down_revision = 'f0f42ffaa27d'
+
+
+def upgrade():
+    op.create_index(op.f('ix__group__readable_by'), 'group', ['readable_by'], unique=False)
```

Comment:
As a general matter of hygiene we should be using `CREATE INDEX CONCURRENTLY` for this. Have a look at previous migrations which added indices.

(Otherwise we lock the group table until the index is built).